conf/app.src

```diff
@@ -0,0 +1,6 @@
+SOURCE_URL=https://github.com/Z-Hub/Z-Push/archive/3b937fb5c79068553198347ffad1f39f2e9d1b51.tar.gz
```

Comment:
You should try with the release, instead of a commit
https://github.com/Z-Hub/Z-Push/releases/tag/2.4.5

Reply:
:/// I will switch to the release, 
I don't why I've done that maybe because their main GIT doesn't provide releases: https://stash.z-hub.io/projects/ZP/repos/z-push/